admin/client/App/index.js

```diff
@@ -21,8 +21,6 @@ Keystone.User = listsByKey[Keystone.userList];
 const { adminPath } = Keystone;
 
 const doRender = () => {
-	const Routes = require('./Routes').default;
```

Comment:
@wmertens Changed it. Like this, it should work properly. We are not fully ES6 compliant anymore which is kind of a pity, but not severe.